examples/graph3d/11_tooltips.html

```diff
@@ -34,10 +39,10 @@
           var z = custom(x,y);
           if (withValue) {
             var value = (y - x);
-            data.add({x:x, y:y, z: z, style:value});
+            data.add({x:x, y:y, z: z, style:value, extra: extra_content[(x*y) % extra_content.length]});
           }
           else {
-            data.add({x:x, y:y, z: z});
+			data.add({x:x, y:y, z: z, extra: extra_content[(x*y) % extra_content.length]});
```

Comment:
Can you please change the ident every where from "tab" to "spaces". (I also don't like it, but ithe the projects standart.)